app/assets/stylesheets/views/_ministers.scss

```diff
@@ -0,0 +1,7 @@
+.ministers-index h2 {
```

Comment:
If you could add something to the commit message about why this styling is needed that'd be great. Also a couple of screenshots in the PR description to show the difference between existing whitehall rendering and this one

Reply:
Done!